---
layout: default
---
<div class="home">
  <div class="materials-wrap">
    <h2 class="module-header">MIT 6.S977/20.S948    Machine Learning Based Therapeutic Design    3-0-3 (G)</h2>


    <h2 class="module-header">Course description</h2>
    <p>Advanced seminar on computational methods in the design and analysis of human therapeutics, including vaccines,  small molecules, biologics, cell based therapies, and synthetic biology approaches. Lectures will present recent results on computational methods on vaccine design, molecular design and optimization drawing upon advances in machine learning.  Recitations will include presentations by students on recent research results related to the computational design of therapeutics and efficacy. Topics include protein design, antibody optimization, vaccine design, small molecule design and characterization, and the engineering of viruses and cell lines for therapeutic effect. Experts from industry and faculty will present their views of the promise of computational approaches, what is working, and what is needed.
    </p>

    <h2 class="module-header">No recitation on February 22nd, 2022</h2>
    <p>
        Per MIT policy, February 22nd, 2022 will be held as Monday's class. We therefore shall not have in-person recitation as usual. Still, please fill in <a href="https://docs.google.com/forms/d/14tOqEeipIZocCmbv-Txpz2ZtFWRQsm6KYIESI-hInHU/viewform?edit_requested=true">mock check-in</a> before 2/22 with questions to ask to Dr. Andrian Veres. 
    </p>

    <h2 class="module-header">Announcement, February 15th, 2022 - PSET1</h2>
    <p>
        <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/6_S977_PSET1.pdf">Problem set 1</a> is due at 11 AM on Marth 15th, 2022. Download <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/pfizer.fasta">pfizer</a> and <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/moderna.fasta">moderna</a> vaccine sequences. Notice that all 1-methyl-3'-pseudouridylyl bases are replaced by thymidines. 
        You can do it yourself or form a 2 or 3-people team. If you decide to form a team, please list all your collaborators. 
        P.S. If you are dying to know more about mRNA vacines against SARS-CoV-2 viruses, you can read New Engliand Journal of Medicine's articles on <a href="https://www.nejm.org/doi/full/10.1056/nejmoa2034577"> Pfizer</a> and <a href="https://www.nejm.org/doi/full/10.1056/nejmoa2035389"> Moderna</a> vaccines and/or FDA Letters of Authorization for <a href="https://www.fda.gov/media/150386/download"> Pfizer</a> and <a href="https://www.fda.gov/media/144636/download"> Moderna</a> vaccines. Doing so is absolutely not necessary. 
    </p>

    <h2 class="module-header">Announcement, February 3rd, 2022</h2>
    <p>Welcome to Machine Learning Based Therapeutics Design! Please complete the <a href="https://docs.google.com/forms/d/1akPByHY6pOsICneBdrRpf46HhclQrnwLs4mIHlDQpkg/edit">first-day survey</a>, whether you're taking the course for credit, registered as a listener, or simply here for fun. If you're taking the course for credit, please <a href="https://docs.google.com/spreadsheets/d/12bvR_S1a4dP2cFnyUgKombYp9EegUMe3TkLRjqnaZ9U/edit#gid=0">sign up for paper presentation.</a>
    </p>

    <h2 class="module-header">Syllabus and schedule</h2>
    <table class="table">
        <th>Date</th><th>Title</th><th>Speaker</th><th>Role</th><th>Affiliation</th><th>Papers</th><th>WebCast Link</th><th>Recitation</th>
        <tr><td>02/03</td><td>A new paradigm for vaccine evaluation, design, and its application to SARS-CoV-2</td><td>David Gifford</td><td>Professor</td><td>MIT</td>
            <td><a href="https://www.cell.com/cell-systems/fulltext/S2405-4712(20)30238-6">Liu2020a</a>
            <a href="https://arxiv.org/abs/2101.10902">Liu2021</a>
            <a href="https://www.biorxiv.org/content/10.1101/2021.05.02.442052v2">Carter2021</a></td>
            <td><a href="https://mit.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=10afbc80-8677-49eb-a000-ae2c012943b4">Link</a></td>
            <td>No recitation</td>
        </tr>

        <tr><td>02/10</td><td>Synthetic, biodegradable lipid nanoparticles for gene therapy and gene editing</td><td>Qiaobing Xu</td><td>Professor</td><td>Tufts University</td>
            <td><a href="https://www.pnas.org/content/118/10/e2020401118">Qiu2021</a>
            <a href="https://www.science.org/doi/10.1126/sciadv.abf1244">Chen2021</a>
            <a href="https://www.science.org/doi/10.1126/sciadv.abb4429">Ma2020</a></td>
            <td>
                <a href="https://mit.zoom.us/rec/share/-lEONPRTp1-13ZbpKoW7LGjqe1fGvCNxa61Dot_1kwhUwZ6Sa-zm30giGinp0uD6.feLDZ96mPrH-_U_G?startTime=1644509162000">Recording</a>
            </td>
            <td>
                <a href="https://docs.google.com/forms/d/16G33b9oX7WmCAlEPg31vDijN75TzZ1va72AO1Q1UEPU/edit">check-in (2/8)</a>
                <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/2_8_2022%20paper%20presentation%20in%206.881_20.S948.pdf">presentation</a></td>
        </tr>

        <tr><td>02/17</td><td>COVID-19: Developing a Vaccine During a Pandemic</td><td>Dan Barouch</td><td>Professor</td><td>Harvard Medical School, Center for Virology and Vaccine Research</td>
            <td><a href="https://pubmed.ncbi.nlm.nih.gov/35102312/">Liu2022</a>
            <a href="https://pubmed.ncbi.nlm.nih.gov/33740454/">Gebre2021</a></td>
            <td><a href="https://mit.zoom.us/rec/share/5orVgUDabZfTB8zysBM1_ArvHbtEOKTqFF5F8aSPTnvfb37DKr7i6girh_vPeG1c.JzlCjK15MH9Qk5Ir?startTime=1645113957000">Recording</a></td>
            <td><a href="https://docs.google.com/forms/d/12kSdXCfE1zKBGFFvnd--izaFwEfGPECS4lzUU1X53us/viewform?edit_requested=true">check-in (2/15)</a>
            <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/2_15_22%20paper%20presentation.pdf">presentation</a></td>
        </tr>
        
        <tr><td>02/24</td><td>Regenerative medicine for Type 1 Diabetes</td><td>Adrian Veres</td><td>Postdoctoral Fellow</td><td>Harvard University</td>
            <td><a href="https://www.nature.com/articles/s41586-019-1168-5">Veres2019</a>
            <a href="https://www.nature.com/articles/s41467-021-23518-w">Yeo2021</a>
            <a href="https://www.cell.com/cell/pdf/S0092-8674(21)01577-4.pdf">Qiu2022</a></td>
            <td><a href="https://mit.zoom.us/rec/share/uAhHBuRufODNJ51zBB4uNDZOhFBqfWTaTnRomsl81Ae7ZPUIvBCITvgR5wH47yaM.4cvd00LQ0LD2NaVw">Recording</a>
                <a href="https://github.com/gifford-lab/mit-6.S977/blob/main/2022_02_24_Veres.pdf.zip">Presentation</a>
            </td>
            <td><a href="https://docs.google.com/forms/d/14tOqEeipIZocCmbv-Txpz2ZtFWRQsm6KYIESI-hInHU/viewform?edit_requested=true">mock check-in</a></td>     
        </tr>

        <tr><td>03/03</td><td>Data-driven small molecule design and synthesis</td><td>Connor Coley</td><td>Professor</td><td>MIT</td>
            <td><a href="https://pubs.acs.org/doi/10.1021/acs.accounts.8b00087">Coley2018</a>
            <a href="https://pubs.rsc.org/en/content/articlelanding/2021/sc/d0sc06805e">Graff2021</a>
            <a href="https://arxiv.org/abs/2110.06389">Gao2021</a></td>      
            <td>Link</td>
            <td>check-in (3/1)</td>
        </tr>

        <tr><td>03/10</td><td>Using computational methods to address druggability and drug discovery challenges</td><td>Jose Duca</td><td>Global Head of Computer-Aided Drug Discovery</td><td>Novartis</td>
            <td><a href="https://www.nature.com/articles/nature18621">Chen2016</a>
            <a href="https://pubs.acs.org/doi/10.1021/acs.jcim.8b00744">Dickson2018</a>
            <a href="https://pubmed.ncbi.nlm.nih.gov/27595330/">Yamada2016</a></td>
            <td>Link</td>
            <td>check-in (3/8)</td>
        </tr>

        <tr><td>03/17</td><td>ML-driven small molecule selection in drug discovery</td><td>Jeremy Jenkins</td><td>US Head, Chemical Biology & Therapeutics</td><td>Novartis</td><td>
            <a href="https://www.nature.com/articles/nrd.2017.232">Schneider2017</a>
            <a href="https://chemrxiv.org/engage/chemrxiv/article-details/60f5d0b888044385cfe22cdc">Godinez2021</a></td>
            <td></td>
        </tr>

        <tr><td>03/24</td><td>Spring Break</td><td>Enjoy your break!</td><td></td><td></td><td></td><td></td></tr>
        <tr><td>03/31</td><td>Synthetic Biology Approaches to Disease Therapeutics</td><td>Ron Weiss</td><td>Professor</td><td>MIT</td><td></td><td></td></tr>
        <tr><td>04/07</td><td>Antibody CDR Design  Using High-Capacity Machine Learning</td><td>Stefan Ewert</td><td>Associate Director</td><td>Novartis</td><td>
            <a href="https://academic.oup.com/bioinformatics/article/36/7/2126/5645171">Liu2020b</a>
            <a href="https://www.biorxiv.org/content/10.1101/2022.01.17.476670v1">Dai2021</a></td>
            <td></td>
        </tr>

        <tr><td>04/14</td><td>Antigen recognition in immunotherapies: TCR and signaling domain engineering</td><td>Michael Birnbaum</td><td>Professor</td><td>MIT</td><td>
            <a href="https://www.biorxiv.org/content/10.1101/2021.07.09.450900v1">Kyung2021</a>
            <a href="https://www.biorxiv.org/content/10.1101/2021.09.18.460796v1">Dobson2021</a>
            <a href="https://www.nature.com/articles/s41592-020-0867-z">Joglekar2021</a></td>
            <td></td>
        </tr>

        <tr><td>04/21</td><td>Using deep learning to decode the genome and 'program' life-changing medicines</td><td>Brendan Frey</td><td>Founder & CEO</td><td>Deep Genomics</td>
            <td>
                <a href="https://www.nature.com/articles/s41525-020-0123-6/">Merico2020</a>
                <a href="https://www.ncbi.nlm.nih.gov/pmc/articles/PMC4362528/">Xiong2015</a>
                <a href="https://www.nature.com/articles/nbt.3300">Alipanahi2015</a>
                <a href="https://www.nature.com/articles/nbt.4233">Wainberg2018</a>
            </td>
            <td></td></tr>

        <tr><td>04/28</td><td>Vaginal microbiome, mucosal immunology, and HIV in sub-Saharan Africa</td><td>Douglas Kwon</td><td>Professor</td><td>Harvard Medical School</td>
            <td>
                <a href="https://pubmed.ncbi.nlm.nih.gov/28087240/">Gosmann2017</a>
                <a href="https://pubmed.ncbi.nlm.nih.gov/30217812/">Ndung'u2018</a>
                <a href="https://www.biorxiv.org/content/10.1101/2021.06.12.448098v1">Bloom2021</a></td>
            </td>
            <td></td></tr>

        <tr><td>05/05</td><td>CRISPR and protein design</td><td>Han Altae-Tran</td><td>PhD student</td><td>Broad Institute of MIT and Harvard</td><td></td><td></td></tr>
    </table>

    <h2 class="module-header">Class meeting times and places</h2>
    <ul>
        <li>Lecture: Thursday 11AM-12:30PM MIT Room 32-155</li>
        <li>Recitation: Tuesday 11AM-12:30PM MIT Room 32-155 (first recitation February 8, 2022)</li>
    </ul>

    <h2 class="module-header">Office hours</h2>
    <div class="materials-item">
        David Gifford (gifford@mit.edu): Office hours by appointment
    </div>
    <div class="materials-item">
       Hyunjin Park (hyunjinp@mit.edu): Monday 3 PM, <a href="https://mit.zoom.us/j/9246873680">Zoom</a>
    </div>

    <h2 class="module-header">Grading</h2>
    <p><p>There will be two problem sets.  Problem Set 1 explores the design of contemporary mRNA COVID-19 vaccines. Problem Set 2 explores the use of deep learning for antibody design.   Students will also present papers during recitation. If you will need special accomodations, please email Student Support Services: s3-support@mit.edu. We will be happy to accomodate!</p>
  </div>
</div>
